src/IdentityServer/Validation/Models/ValidatedAuthorizeRequest.cs

```diff
@@ -221,9 +221,11 @@ public class ValidatedAuthorizeRequest : ValidatedRequest
     public string? PushedAuthorizationReferenceValue { get; set; }
 
     /// <summary>
-    /// Gets a value indicator if the authorize request's parameters where pushed using PAR. 
+    /// Gets or sets a value indicating the context in which authorization
+    /// validation is occurring (the PAR endpoint or the authorize endpoint with
+    /// or without pushed parameters).
     /// </summary>
-    public bool IsPushedAuthorizationRequest => PushedAuthorizationReferenceValue is not null;
+    public AuthorizeRequestType AuthorizeRequestType { get; set; }
```

Comment:
Maybe make it overly explicit that it's set to `Authorize` by default?

Reply:
Meh, I now see this is used in many places... we can just rely upon the default enum values, I suppose.